src/control/cmd/daos/attribute.go

```diff
@@ -88,9 +98,6 @@ func listDaosAttributes(hdl C.daos_handle_t, at attrType, verbose bool) ([]*attr
 	attrNames := []string{}
 	expectedSize = totalSize
 	buf := C.malloc(totalSize)
-	if buf == nil {
-		return nil, errors.New("failed to malloc buf")
-	}
```

Comment:
If `C.malloc()` fails it [will panic](https://pkg.go.dev/cmd/cgo#hdr-Go_references_to_C):
> As a special case, C.malloc does not call the C library malloc directly but instead calls a Go helper function that wraps the C library malloc but guarantees never to return nil. If C's malloc indicates out of memory, the helper function crashes the program, like when Go itself runs out of memory. Because C.malloc cannot fail, it has no two-result form that returns errno.